Finmer.Game/Gameplay/SceneCombat2.cs

```diff
@@ -641,12 +660,17 @@ private void ShowUIGrappling(GameUI ui)
                 // Player is being pinned
                 ui.Instruction = $"You're being pinned down by {m_Player.GrapplingWith.Character.Name}! What will you do?";
 
-                ui.AddButton(new ChoiceButtonModel
+                // If player cannot start the grapple against larger target, make sure they cannot reverse it either.
+                var grapple_targets = GetViableGrappleTargets(m_Player.GrapplingWith);
```

Comment:
This won't do what you want it to do. Read the logic step by step:

- Get the participant that the player is currently grappling with
- Get a list of participants that that specific participant could initiate a grapple with
- If there are any, show the Reverse button

That makes no sense. Rather, we want the Reverse button to be shown if the _player_ could theoretically initiate a grapple with their opponent, since that is the condition for the Grapple button to be shown ordinarily.